contrib/mkimage-yum.sh

```diff
@@ -56,13 +67,24 @@ mknod -m 666 "$target"/dev/urandom c 1 9
 mknod -m 666 "$target"/dev/zero c 1 5
 
 # amazon linux yum will fail without vars set
-if [ -d /etc/yum/vars ]; then
-	mkdir -p -m 755 "$target"/etc/yum
-	cp -a /etc/yum/vars "$target"/etc/yum/
+if [ -d /etc/yum/vars ];
```

Comment:
For a cleaner history, might be better to leave that for a separate PR, but I'll leave that up to @jfrazelle